protocol/rpcprovider/rpcprovider.go

```diff
@@ -110,6 +110,7 @@ type rpcProviderStartOptions struct {
 	healthCheckMetricsOptions *rpcProviderHealthCheckMetricsOptions
 	staticProvider            bool
 	staticSpecPath            string
+	relayLoadLimit            uint
```

Comment:
In Golang uint, without specifying the size, it will default to the operating system defaults. 
we don't want different behaviors in different environments.